ambry-frontend/src/main/java/com.github.ambry.frontend/AmbrySecurityService.java

```diff
@@ -122,8 +123,9 @@ public AmbrySecurityService(FrontendConfig frontendConfig, FrontendMetrics front
               } else {
                 responseChannel.setHeader(RestUtils.Headers.LAST_MODIFIED,
                     new Date(blobInfo.getBlobProperties().getCreationTimeInMs()));
-                setGetBlobResponseHeaders(responseChannel, blobInfo);
+                setGetBlobResponseHeaders(blobInfo, restRequest, responseChannel);
               }
+              responseChannel.setHeader(RestUtils.Headers.ACCEPT_RANGES, "bytes");
```

Comment:
This should be provided to signify that a certain resource supports range requests. So, right now, I send it on responses to all GET requests without specified subresources.

```
      The Accept-Ranges response-header field allows the server to
      indicate its acceptance of range requests for a resource:
          Accept-Ranges     = "Accept-Ranges" ":" acceptable-ranges
          acceptable-ranges = 1#range-unit | "none"
      Origin servers that accept byte-range requests MAY send
          Accept-Ranges: bytes
      but are not required to do so. Clients MAY generate byte-range
      requests without having received this header for the resource
      involved. Range units are defined in section 3.12.
      Servers that do not accept any kind of range request for a
      resource MAY send
          Accept-Ranges: none
      to advise the client not to attempt a range request.
```